docs/empower.rst

```diff
@@ -0,0 +1,47 @@
+Empower
+=======
+
+********
+Overview
+********
+
+The Empower class allows you to interact with the Empower API. Documentation for the Empower API can be found
+in their `GitHub <https://github.com/getempower/api-documentation/blob/master/README.md>`_ repo.
+
+.. note::
```

Comment:
Can you change this note to include information about authentication, and then move this note into the overview? Just to maintain consistency with how other connectors docs work.